.github/scripts/addComponentLabel.js

```diff
@@ -21,10 +21,16 @@ module.exports = async ({ github, context }) => {
   if (whichComponentRegexMatch) {
     const componentsString = (whichComponentRegexMatch[1] || "").trim();
 
-    if (componentsString !== "N/A" && componentsString !== "Unknown / Not Sure") {
-      const componentsList = componentsString
-        .split(", ")
-        .map((component) => `c-${component}`.replace(" ", "-").toLowerCase());
+    // Filter out "N/A" and "Unknown / Not Sure" so we don't create labels for those.
+    const filteredComponents = componentsString
+      .split(",")
+      .map((component) => component.trim())
+      .filter((component) => {
+        return component !== "N/A" && component !== "Unknown / Not Sure" && component !== "N/A, Unknown / Not Sure";
```

Comment:
It might help to move the fixed labels to local constants to DRY things up a bit.